protocols/imap/src/main/java/org/apache/james/imap/message/request/ListRequest.java

```diff
@@ -46,11 +68,25 @@ public final String getMailboxPattern() {
         return mailboxPattern;
     }
 
+    public EnumSet<ListSelectOption> getSelectOptions() {
+        return selectOptions;
+    }
+
+    public EnumSet<ListReturnOption> getReturnOptions() {
+        return returnOptions;
+    }
+
+    public final boolean selectSubscribed() {
```

Comment:
```suggestion
    public boolean isSelectSubscribed() {
```

 -> No need "final"
 -> Boolean getters starts by "is..." ?